README.md

```diff
@@ -1,8 +1,14 @@
+{{< site-region region="us3,eu,gov" >}}
+<div class="alert alert-warning">
+  This integration is only supported in the US East (us-east-1) region.
+</div>
+{{< /site-region >}}
+
 # Datadog-AWS CloudFormation
 
 [AWS CloudFormation][1] gives you templates to describe, configure, and provision all of the AWS resources in your environment at once. The Datadog-AWS CloudFormation Resources allow you to interact with the supported Datadog resources.
```

Comment:
```suggestion
[AWS CloudFormation][1] gives you templates to describe, configure, and provision all of the AWS resources in your environment at once. The Datadog-AWS CloudFormation Resources allow you to interact with the supported Datadog resources, send resources to any Datadog datacenter, and privately register an extension in any region with Datadog resources.
```

Reply:
@zippolyte I added your comments to the site partial in this introductory paragraph. Does this content logically make sense?